en/setting-up-the-app-in-shopify.md

```diff
@@ -0,0 +1,116 @@
+
+   ![Alt text](../img/Screenshot%202022-08-28%20at%2010.51.06.png?raw=true "Sakurabook Dashboard")
+
+9. Now we refer to [Usage Guides](usage-guides.md) where you can find details about managing bookings on the dashboard, setting up and changing options of your staff, rental spaces, and services, and changing the location of your business and all the other staff.
+
+10. The very last step on our Sakurabook Admin Setup guide is a selection of your Billing plan. Navigate to Settings (top menu) and click on the `Manage plan` button under the Billing section. You've got several options, see details about [Billing plans](billing-plans.md).
+
+    📌 Each of the billing plans is limited by the number of paid bookings you can create per month/year. For example, the **Standard plan** with a monthly subscription is only for **$19.90** and you can create **6.000** paid bookings. Moreover, when you subscribe for the first time, you'll get a **14 day** free trial. You can test everything and decide if Sakurabook is the right app for you.
+
+    ![Alt text](../img/?raw=true "Sakurabook Billing")
+
+## Sakurabook Booking Wizard Setup
+
+Sakurabook app is installed in your Shopify store and you've successfully finished the previous [guide](#sakurabook-booking-wizard-setup). Now let's have a look at how to add the Sakurabook Booking Wizard into your customer-facing storefront, also called Shopify Theme.
+
+📌 There is one prerequisite to be able to add Booking Wizard into your storefront. You need to have an Online Store sales channel installed in your store.
+
+1. Go to **Online Store** in the left main menu under **Sales channels** section and click on **Theme**. It depends on what theme you have selected, but you should see the **Current theme** section. Click on the `Customize` button.
+
+   ![Alt text](../img/Screenshot%202022-08-28%20at%2010.36.48.png?raw=true "Shopify Theme Customize")
+
+2. Now you find yourself in the Shopify Theme customization tool where you can fully set up your online store. In the top middle selection box with Templates, search the Products section which is the place where **Sakurabook App block** (Booking Wizard) needs to be placed.
+
+   ![Alt text](../img/Screenshot%202022-08-28%20at%2010.37.12.png?raw=true "Shopify Theme Products")
+
+3. In the left main menu, you should now see the page sections. Drop down the **Product information** section and clock on the `Add block` button. A new window pops up and it contains two lists, THEME BLOCKS, and APPS. Under the APPS list, you should see **Booking Wizard** from **sakurabook** app. Click on it and it will be added to your theme. The app should load itself even in the customization tool. All is now set. Click on the three dots menu in the top left corner of the screen and go to **View** to check it out in your customer-facing storefront.
```

Comment:
```suggestion
3. In the left main menu, you should now see the page sections. Drop down the **Product information** section and clock on the `Add block` button. A new window pops up and it contains two lists - THEME BLOCKS and APPS. Under the APPS list, you should see **Booking Wizard** from **Sakurabook** app. Click on it and it will be added to your theme. The app should load itself even in the customization tool. All is now set. Click on the three dots menu in the top left corner of the screen and go to **View** to check it out in your customer-facing storefront.
```